.circleci/config.yml

```diff
@@ -368,6 +370,7 @@ jobs:
       - run:
           name: integration test
           command: sudo -E test/test_runner.sh $TEST_ARGS 2>&1 | tee -a /tmp/testout
+          no_output_timeout: 30m
```

Comment:
Thanks for adding this (saw this "no output for 10 minutes" a lot recently), but is it related to the change in this PR ("Upload bundle in separate CI step")?

If not really, I'd prefer to have this one as a separate commit (same PR is totally fine)

Reply:
Yep, I added it as second commit. I'm not sure if this increased timeout is necessary because of actual changes to the integration tests or hardware changes in CircleCI. It _feels_ that their machines slowed down in the past weeks, but I might be wrong.

Reply:
Thanks!

Yes, I have the same feeling (slower machines, reduced concurrency).